docs/release_notes.rst

```diff
@@ -35,6 +35,7 @@ New Features
 * Add support for `property_decorators` config to ignore D401.
 * Add support for Python 3.10 (#554).
 * Replace D10X errors with D419 if docstring exists but is empty (#559).
+* Add support for Sphinx-style parameter descriptions to D417.
```

Comment:
@benji-york could you please move this to the latest version?

Reply:
@benji-york any objects if I have a go at updating this?

Reply:
> @benji-york any objections if I have a go at updating this?

That'd be great, @tomghc!

(Sorry, I let this fall off my radar.  If I can be of any service, let me know.)

Reply:
@benji-york Hey, I was looking to help finish updating this to the current version so it could be pushed. Is there any way I could aid in development?

Reply:
I'm happy for anyone to do anything on this.  I'm not sure what @samj1912 means above by "move this to the latest version".  Perhaps rebase?

Reply:
The automated rebase was successful.  All tests passed.  Now the review status has been reset, so this needs another review by a maintainer and if still acceptable (which it should be because nothing has changed), it needs merging by a maintainer.

Reply:
Sounds great! Apologies for any inconvenience that I may have caused. I appreciate the work you put in!